examples/plot_new_metrics.py

```diff
@@ -231,14 +230,20 @@ def occupation_transform(occ_string):
 # class. Let us construct an instance of this class, and then look at
 # its capabilities:
 
-fbeta_06 = functools.partial(skm.fbeta_score, beta=0.6)
+fbeta_06 = functools.partial(skm.fbeta_score, beta=0.6, zero_division=1)
```

Comment:
This is the only "real" change other than the style fixes, right?

Reply:
Confirmed offline that this is true. Thanks @alliesaizan!

Reply:
I'm happy with this. Ideally all the formatting changes would be a separate PR...